### PR TITLE
#1959 - Staging sims prod environment -Part 1

### PR DESCRIPTION
**Pull request by guru-aot**
Changes as part of this PR:
- Patroni deployment is changed, going forward we are not building in tools namespace and deploying them. We use the reference as per sample shared by BCGOV https://github.com/bcgov/patroni-postgres-container/blob/master/samples/deploy.yaml for further deployments.
- Network Policies removed which are not used.
- Role Bindings  for github actions will be added as part of init-oc command.
- Removed the secrets that are not used and removed the obselete methods
  - WORKFLOW_SA_USER_NAME
  - WORKFLOW_SA_PASSWORD
  - SIMS_API_CLIENT_ID
  - SIMS_API_CLIENT_SECRET

### CODE REVIEW

File: sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts [hunk 1]
```diff
@@ -99,24 +99,6 @@ export class KeycloakService {
     return this.getKeyCloakToken(payload);
   }
 
-  /**
-   * Retrieve an token from Keycloak using a provided client id and client secret.
-   * @param clientSecret Client secret.
-   * @param clientId Client ID.
-   * @returns token
-   */
-  public async getTokenFromClientSecret(
```
Comment (ann-aot) [inline sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts:108]: why was this removed

**Review by ann-aot — APPROVED** [review]
👍 good work @guru-aot

File: sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts (hunk 1 above)
Comment (guru-aot) [reply sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts:108]: Yes it was obsolete and do not have any reference

Comment (andrewsignori-aot) [reply sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts:108]: @ann-aot it was not for form.io but it was no longer used.

File: devops/openshift/database/patroni-deploy.yml [hunk 2]
```diff
@@ -186,16 +215,21 @@ parameters:
...
+- name: IMAGE_NAME
+  description: |
+    The Patroni image stream name
+  value: patroni-postgres
+- name: IMAGE_TAG
   description: |
-    Patroni ImageTag
-  value: patroni:12-latest
+    The image tag used to specify which image you would like deployed.
+    Don't use `latest`.
```
Comment (andrewsignori-aot) [inline devops/openshift/database/patroni-deploy.yml:231]: The comment is confusing 😄 
"Don't use `latest`" and then we use "12.4-latest".

Comment (guru-aot) [reply devops/openshift/database/patroni-deploy.yml:231]: :) Copy paste from the repo sample i got from bcgov. I will remove that line, might be an old comment i guess.

File: sources/packages/backend/apps/api/src/services/auth/tokens.service.ts
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/api/src/services/auth/tokens.service.ts:1]: Thanks for doing the cleanup 😉

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the cleanups and updating Patroni. Looks good 👍

**Review by dheepak-aot — APPROVED** [review]
Great work with patroni and thanks for the clean up. 👍
